### ISSUE TITLE
Peer Review for Patrick

### ISSUE THREAD

**Issue by jwrudolf** [amromeo/api_r2020#18]
@pcmathias 

Hi Patrick.

Took a spin through your slides, love 'em.

Idle thoughts below, feel free to disregard any & all.

(slide #)

(6) - You mention the Duke scandal in your notes, just curious if you are going to unpack that story more
(25) - You mentioned factors.  Are you going to unpack verbally in more detail at this point?
(28) - Should we print the data dictionary as a full page for attendees to work with?
(29/30) - Since I teed them up with Janitor, will you be sure to call out we are using a function from a package we installed in lesson 1?
(31) - You might want to call out that you want them to add a third variable, some might interpret as 3 more variables.
(38) - Wondering if the audience may need a little more clues to do this on their own early in the day (e.g. a nudge to the right argument or some sample code to build on).  Maybe I am underestimating them.

**Comment by jwrudolf** [comment]
One last thought, maybe a quick goals/objectives slide.

**Comment by pcmathias** [comment]
(6) I copied a few slides from Stephan's course so notes came over. It would be great to tell the Baggerly story but I don't think I'll have time.
(25) I was planning to say factors = categorical variables. It would be good to show the distinction in a summary but I'm not confident I'll have time.
(28) This would be helpful. Is @skadauke on point for printing?
(29/30) Yep I should call this out. The little janitor icon should be my cue but I'm sure I'll forget.
(31) I updated to clarify language.
(38) Good thought. I added code stubs for them.

I added a goals slide as well - thanks for the input!
